### ISSUE TITLE
2025.02.05 - #23

### ISSUE THREAD

**Issue by jy6757** [changh95/WeeklySpatialAI#25]

**Comment by chae2chae** [comment]
### [Pi0](https://www.physicalintelligence.company/)
- https://github.com/Physical-Intelligence/openpi.git 

- code and model weights for running the base pre-trained π model
- several checkpoints that are fine-tuned for a few simple tasks on a few widely available platforms such as [[ALOHA](https://tonyzhaozh.github.io/aloha/)](https://tonyzhaozh.github.io/aloha/) and [[DROID](https://droid-dataset.github.io/)](https://droid-dataset.github.io/)
- example code to run inference on several real-world and simulated robot platforms
- code for fine-tuning the base π model for your own tasks and platforms


### AnyTeleop: A General Vision-Based Dexterous Robot Arm-Hand Teleoperation System

- https://arxiv.org/abs/2307.04577
- https://github.com/dexsuite/dex-retargeting

![Image](https://github.com/user-attachments/assets/cb68d41b-9077-4bc4-96da-de1f5de99290)


### VideoDex: Learning Dexterity from Internet videos [CoRL 2022]
- https://arxiv.org/pdf/2212.04498

![Image](https://github.com/user-attachments/assets/1705e83d-6bc6-4a9c-a632-8c030e421b79)

**Comment by changh95** [comment]
# Object SLAM paper list

# Semantic SLAM paper list (dense mapping)

# SLAM with Scene graph representation paper list

- Long-Term Human Trajectory Prediction using 3D Dynamic Scene Graphs
- Clio: Real-time Task-Driven Open-Set 3D Scene Graphs
- Task and Motion Planning in Hierarchical 3D Scene Graphs
- [[Indoor and outdoor 3d scene graph generation via language-enabled spatial ontologies](https://ieeexplore.ieee.org/abstract/document/10487851/)](https://ieeexplore.ieee.org/abstract/document/10487851/)
- https://arxiv.org/abs/2402.13817
- [[Foundations of spatial perception for robotics: Hierarchical representations and real-time systems](https://journals.sagepub.com/doi/abs/10.1177/02783649241229725)](https://journals.sagepub.com/doi/abs/10.1177/02783649241229725)
- [[Kimera2: Robust and accurate metric-semantic SLAM in the real world](https://link.springer.com/chapter/10.1007/978-3-031-63596-0_8)](https://link.springer.com/chapter/10.1007/978-3-031-63596-0_8)
- [[Hydra-multi: Collaborative online construction of 3d scene graphs with multi-robot teams](https://ieeexplore.ieee.org/abstract/document/10341838/)](https://ieeexplore.ieee.org/abstract/document/10341838/)
- [[Hierarchical representations and explicit memory: Learning effective navigation policies on 3d scene graphs using graph neural networks](https://ieeexplore.ieee.org/abstract/document/9812179/)](https://ieeexplore.ieee.org/abstract/document/9812179/)
- [[Hydra: A real-time spatial perception system for 3D scene graph construction and optimization](https://arxiv.org/abs/2201.13360)](https://arxiv.org/abs/2201.13360)
- [[Kimera-multi: Robust, distributed, dense metric-semantic slam for multi-robot systems](https://ieeexplore.ieee.org/abstract/document/9686955/)](https://ieeexplore.ieee.org/abstract/document/9686955/)
- [[Kimera: From SLAM to spatial perception with 3D dynamic scene graphs](https://journals.sagepub.com/doi/abs/10.1177/02783649211056674)](https://journals.sagepub.com/doi/abs/10.1177/02783649211056674)
- [[Kimera: an open-source library for real-time metric-semantic localization and mapping](https://ieeexplore.ieee.org/abstract/document/9196885/)](https://ieeexplore.ieee.org/abstract/document/9196885/)
- [[3D dynamic scene graphs: Actionable spatial perception with places, objects, and humans](https://arxiv.org/abs/2002.06289)](https://arxiv.org/abs/2002.06289)
- [**[3D Scene Graph: A Structure for Unified Semantics, 3D Space, and Camera](https://www.semanticscholar.org/paper/77b4f542ea56b02804672790b6482df400428c95)**](https://www.semanticscholar.org/paper/77b4f542ea56b02804672790b6482df400428c95)

- SceneGraphFusion: Incremental 3D Scene Graph Prediction from RGB-D Sequences
- [**[S-Graphs+: Real-Time Localization and Mapping Leveraging Hierarchical Representations](https://www.semanticscholar.org/paper/3d2af0aa792535e4f8a943274091de222fe5c1d8)**](https://www.semanticscholar.org/paper/3d2af0aa792535e4f8a943274091de222fe5c1d8)
- ConceptGraphs
- [**[SayPlan: Grounding Large Language Models using 3D Scene Graphs for Scalable Task Planning](https://www.semanticscholar.org/paper/3843469976bd895851bfa08c8208350745bf649f)**](https://www.semanticscholar.org/paper/3843469976bd895851bfa08c8208350745bf649f)
- [**[Open3DSG: Open-Vocabulary 3D Scene Graphs from Point Clouds with Queryable Objects and Open-Set Relationships](https://www.semanticscholar.org/paper/5f0bb5267aef221e99c05c49dd30d488f94636dd)**](https://www.semanticscholar.org/paper/5f0bb5267aef221e99c05c49dd30d488f94636dd)
- Hierarchical Open-Vocabulary 3D Scene Graphs for Language-Grounded Robot Navigation
- LM-Nav: Robotic Navigation with Large Pre-Trained Models of Language, Vision, and Action
- CLIP-Fields: Weakly Supervised Semantic Fields for Robotic Memory
- [**[Graph R-CNN for Scene Graph Generation](https://www.semanticscholar.org/paper/1fad7fe0a7a90a8470a0688ad26bab6ceb8a85b7)**](https://www.semanticscholar.org/paper/1fad7fe0a7a90a8470a0688ad26bab6ceb8a85b7)

**Comment by changh95** [comment]
네트워크상의 문제로 @jy6757 의 발표는 다음주에 다시 진행됩니다! 😸
